force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -192,13 +245,24 @@ public with sharing class ClassesObjectsMethods {
     public static String getDayOfWeek(DateTime dt) {
         // Define a map of day of the week as a key and day number as the value
         // Monday = 1, Tuesday = 2, etc.
+        Map<String, Integer> dayOfWeekMap = new Map<String, Integer>{
+            'Monday' => 1,
+            'Tuesday' => 2,
+            'Wednesday' => 3,
+            'Thursday' => 4,
+            'Friday' => 5,
+            'Saturday' => 6,
+            'Sunday' => 7
+        };
 
         // Get the day name using .format('EEEE');
+        String dayName = dt.format('EEEE');
 
         // Get the day number from the map using the day name
+        Integer dayNumber = dayOfWeekMap.get(dayName);
 
         // Return the result as a string
-        return null; // Replace null with the variable you used to store the result
+        return dayNumber != null ? dayNumber.toString() : null; // Replace null with the variable you used to store the result
```

Comment:
These instructions are intentionally somewhat verbose for practice. However, can you do this in one like by using the Datetime.format() method?